aiven/client/cli.py

```diff
@@ -4101,8 +4101,19 @@ def _get_service_type_user_config_schema(self, project: str, service_type: str)
 
         return service_def["user_config_schema"]
 
+    def _get_service_versions(self) -> Sequence[Mapping[str, Any]]:
+        default_project = self.config.get("default_project")
```

Comment:
Tests seem to be failing as default project may not exist. We may need to check that the attribute exist first.
Also, should we consider adding an argument to pass a project instead of switching to check versions?

Reply:
`'AivenCLI' object has no attribute 'config'` :exploding_head:  I thought that was not possible.  I'll fix that.